# These are all of the potential parameters that we can set in the groundstation

sail_lookup_table_wind_angles:    [0,    45,   90,   135,   180,   225,   270,   315,   360]
sail_lookup_table_sail_positions: [70.0, 50.0, 30.0, 10.0,  0.0,   10.0,  30.0,  50.0,  70.0]

perform_forced_jibe_instead_of_tack: False,
waypoint_accuracy: 5    # meters
tack_distance: 100 # rough distance to sail before changing tacks based on dynamic tacking system. Ask Chris for more info

rudder_p_gain: 1
rudder_i_gain: 0.3
rudder_d_gain: 0.2
rudder_n_gain: 1

heading_p_gain: 1
heading_i_gain: 0
heading_d_gain: 0
heading_n_gain: 1

rudder_hard_over: 25
no_sail_zone_size: 100
autopilot_refresh_rate: 30

jibe_tolerance: 3   # how close to jibe angle to stop steering around and start sailing along the jibe line
tack_tolerance: 3   # how close to tacking angle to stop steering around and start sailing along the tack line

min_sail_angle: 0
max_sail_angle: 90

min_rudder_angle: -25
max_rudder_angle: 25